lecture/views.py

```
@@ -1,49 +1,248 @@
+        context["course"] = self.course
+        return context
+
+class CreateContentView(LoginRequiredMixin, FormView):
+    template_name = "lecture/create_content.html"
+
+    def get_form_class(self):
+        """ Dynamically choose the form based on the 'type' parameter """
+        type = self.kwargs.get('type')
+        if type == "attachment":
+            return LecturePDFForm
+        elif type == "video":
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)